cylc/flow/scripts/broadcast.py

```diff
@@ -318,6 +318,17 @@ def get_option_parser() -> COP:
              "the broadcast config structure in raw Python form.",
         action="store_true", default=False, dest="raw")
 
+    parser.add_option(
+        '--json',
+        help=(
+            "With -d/--display or -k/--display-task, write out "
+            "the broadcast config structure in JSON format."
+        ),
+        action='store_true',
+        default=False,
+        dest='json',
+    )
```

Comment:
In an attempt to standardise the Cylc CLI options, can we have `--format=json`, `-t=json`.

Context: https://github.com/cylc/cylc-flow/issues/2972